### PR TITLE
Feat-Single video page

### PR DESCRIPTION
**Pull request by krishanu2911**
Added the single video page with help of useParams hook
-Can add the video to the playlist
-like and watch later feature will come further pr

https://user-images.githubusercontent.com/75560307/161695165-402ea259-e6b8-43d6-86fe-b6c9c27ef114.mp4

### CODE REVIEW

**Review by shubhamkumar648 — APPROVED** [review]
code look neat and clean, as such no changes are required you can merge

File: src/pages/SingleVideoPage/SingleVideoPage.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            width="560"
+            height="315"
+            src={`https://www.youtube.com/embed/${videoId}`}
+            title="YouTube video player"
+            frameBorder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowfullScreen
+            className="margin-bottom"
+            />
+          <footer>
```
Comment (raman1999) [inline src/pages/SingleVideoPage/SingleVideoPage.js:35]: can use react player here for handling responsiveness

**Review by raman1999 — APPROVED** [review]
code looks clean to me, good naming convention used

File: src/pages/SingleVideoPage/SingleVideoPage.js (hunk 1 above)
Comment (krishanu2911) [reply src/pages/SingleVideoPage/SingleVideoPage.js:35]: Thank you Raman for the suggestion will implement it in coming Pr

**Pull request merged**